jwst/resample/resample_step.py

```diff
@@ -1,5 +1,3 @@
-import logging
```

Comment:
This seems to have been lost and is needed in order for the code to be run from the command line

Reply:
Interesting.  I original had it in there, but flake8 suggested I remove it.  Hmm.

Reply:
Right, normally `logging` does not need to be imported as one uses the built in `self.log` of `Step`, but here I also need it in a class method, which does not know anything about `self`.  Not sure if there's a more elegant way around this.

Added a commit to put it back.